src/rpc/mining.cpp

```diff
@@ -530,6 +532,14 @@ returns JSON if pblockOut is NULL
 pblockOut -A copy of the block if not NULL
 */
 
+bool forceTemplateRecalc = false;
+// force block template recalculation
```

Comment:
it this needs cs_main then maybe 

+bool forcTemplateRecalc = false GUARDED_BY(cs_main)

Reply:
oops, that will probably give compile errors on mac. should be

+bool forceTemplateRecalc GUARDED_BY(cs_main) = false;